src/Cmgmyr/Messenger/Traits/Messagable.php

```diff
@@ -61,6 +62,16 @@ public function newThreadsCount()
     }
 
     /**
+     * Returns the new messages count for user.
+     *
+     * @return int
+     */
+    public function newIncomingMessagesCount()
```

Comment:
Maybe it will be better to name it `unreadMessagesCount()`?

Reply:
I think the idea with the word `unread` instead of `new` is good, but that is not consistent with `threadsWithNewMessages` as it uses `new` for `unread`. I put `incoming` there to be clear that we are speaking about incoming messages: we are excluding outbound messages from the scope. My perfect suggestion would be: `unreadIncomingMessagesCount` ... but it's probably too long :)
@a-komarev @cmgmyr what do you think?

Reply:
Why you want to have `Incoming` word in method name? Is there is a need to count sent messages? It's looks redunant for me.

Reply:
I can rebase it if everyone is ok with the name `unreadMessagesCount`?

Reply:
I'm good with `unreadMessagesCount()`. I'd actually like to replace the other `new` references with `unread` since it makes it a little more clear. We'll leave that for another day though.

Reply:
We could replace it with temporary fallback. Make `unread*` method which will be called in old method, and old method should be marked in docblock as `@deprecated`

Reply:
Yup, definitely agreed. Just wanted to note the preferred way moving forward. Not in a rush to update the current API.